src/transformers/models/deta/modeling_deta.py

```diff
@@ -1040,6 +1041,7 @@ class DetaEncoder(DetaPreTrainedModel):
 
     def __init__(self, config: DetaConfig):
         super().__init__(config)
+        self.gradient_checkpointing = False
```

Comment:
I think this shouldn't be here?

```suggestion
```

Reply:
I thought you were right. However, if you see the test CI if fails due to encoder also has override from the class PreTrainedModel.
@amyeroberts Can you confirm this? I might need to revert the last chanage!

Reply:
Yep - you're right, my bad!